src/styles/virtualkeyboard/style.qml

```diff
@@ -55,8 +55,8 @@ KeyboardStyle {
         inputLocaleIndicatorHighlightTimer.restart()
     }
 
-    keyboardDesignWidth: size.ratio(854)//N9
-    keyboardDesignHeight: size.ratio(480)
+    keyboardDesignWidth: size.dp(854)//N9
```

Comment:
If I remember correctly, you wanted to use PinePhone dimensions  1440 x 720

Reply:
yes...i will fix it